sno/working_copy/mysql_adapter.py

```diff
@@ -0,0 +1,222 @@
+from sno import crs_util
+from sno.schema import Schema, ColumnSchema
+
+from sqlalchemy.dialects.mysql.base import MySQLIdentifierPreparer, MySQLDialect
+
+
+_PREPARER = MySQLIdentifierPreparer(MySQLDialect())
+
+
+V2_TYPE_TO_MYSQL_TYPE = {
+    "boolean": "bit(1)",
```

Comment:
I'm a good 95% indifferent to the storage implications of these tiny columns - I chose BIT because 1 bit is the same as 1 boolean, at least in terms of its state space. Of course, we can use a different type if needed: can you spell out the compatibility issue you are trying to avoid here?